src/modules/Settings/selectors.js

```diff
@@ -125,9 +125,15 @@ export const getPluginInfo = (state: State, type: string): EdgeCurrencyInfo => {
   return currencyInfo
 }
 
-export const getSupportedWalletTypes = (state: State) => {
-  const allCurrencyInfos = getPlugins(state).allCurrencyInfos
-
+export type SupportedWalletTypes = {
```

Comment:
Please look at src/util/CurrencyCodeHelpers.js, which already has this exact logic.